elvee.c

```diff
@@ -455,3 +459,71 @@ void timestamp()
     month = (strstr(month_names, month_name) - month_names) / 4 + 1;
     printf("%04hu-%02hu-%02hu %02hu:%02hu:%02hu\n", year, month, day, hour, min, sec);
 }
+
+// Adapted from https://docs.microsoft.com/en-us/archive/blogs/twistylittlepassagesallalike/everyone-quotes-command-line-arguments-the-wrong-way
+// See issue #1
+// The returned pointer is a new allocation, but leaking it is fine.
```

Comment:
My detailed comment about addressing this leak [somehow disappeared from my last review](https://github.com/atifaziz/elvee/pull/2#pullrequestreview-367178805). Very odd.